quickwit-cluster/Cargo.toml

```diff
@@ -14,16 +14,15 @@ anyhow = "1.0"
 async-trait = "0.1"
 flume = "0.10"
 quickwit-common = { version = "0.2.1", path = "../quickwit-common"}
-quickwit-swim = { version = "0.2.1", path = "../quickwit-swim" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 thiserror = "1.0"
 tokio = { version = "1.7", features = [ "full" ] }
 tokio-stream = { version = "0.1", features = [ "sync" ] }
 tracing = "0.1.29"
 uuid = "0.8"
-
 quickwit-proto = { version = "0.2.1", path = "../quickwit-proto"}
+scuttlebutt = { git = "https://github.com/quickwit-oss/scuttlebutt", branch="main" }
```

Comment:
you need a version here, otherwise we will not be able to publish quickwit to Cargo. Also I'd prefer to have rev instead of branch. You can have all of these coexist I believe.
`git, rev, version`